packages/plugins/graphql-executor/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@envelop/graphql-executor",
+  "version": "0.0.0",
+  "description": "A customizable GraphQL Spec compliant Executor",
+  "license": "MIT",
+  "author": "Saihajpreet Singh <saihajpreet.singh@gmail.com> (https://saihaj.dev/)",
+  "sideEffects": false,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/n1ru4l/envelop.git",
+    "directory": "packages/plugins/graphql-executor"
+  },
+  "main": "dist/cjs/index.js",
+  "module": "dist/esm/index.js",
+  "exports": {
+    ".": {
+      "require": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/cjs/index.js"
+      },
+      "import": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      },
+      "default": {
+        "types": "./dist/typings/index.d.ts",
+        "default": "./dist/esm/index.js"
+      }
+    },
+    "./*": {
+      "require": {
+        "types": "./dist/typings/*.d.ts",
+        "default": "./dist/cjs/*.js"
+      },
+      "import": {
+        "types": "./dist/typings/*.d.ts",
+        "default": "./dist/esm/*.js"
+      },
+      "default": {
+        "types": "./dist/typings/*.d.ts",
+        "default": "./dist/esm/*.js"
+      }
+    },
+    "./package.json": "./package.json"
+  },
+  "typings": "dist/typings/index.d.ts",
+  "typescript": {
+    "definition": "dist/typings/index.d.ts"
+  },
+  "scripts": {
+    "test": "jest",
+    "prepack": "bob prepack"
+  },
+  "dependencies": {
+    "graphql-executor": "^0.0.23"
+  },
+  "devDependencies": {
+    "bob-the-bundler": "1.5.1",
+    "graphql": "16.5.0",
+    "typescript": "4.4.4"
+  },
+  "peerDependencies": {
+    "@envelop/core": "^1.6.1",
+    "graphql": "^14.0.0 || ^15.0.0 || ^16.0.0"
+  },
+  "buildOptions": {
+    "input": "./src/index.ts"
+  },
+  "publishConfig": {
+    "directory": "dist",
+    "access": "public"
+  }
```

Comment:
`"type"`: "module"` is missing in this `package.json` - that is why the integrity check fails